lib/plugins/creative.js

```diff
@@ -17,11 +17,6 @@ function inject (bot, { version }) {
   }
 
   const creativeSlotsUpdates = []
-  bot._client.on('set_slot', ({ windowId, slot, item }) => {
```

Comment:
Why

Reply:
we don’t need it, below I replaced the check that used this with just waiting for the inventory event, which is probably better anyway since we can be sure that the inventory received the item